src/components/value-displays/value-label/value-label.test.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { render, screen } from "~/tests/testing-library";
+import { ValueLabel } from "./value-label";
+import userEvent from "@testing-library/user-event";
+import { vi } from "vitest";
+import { Label, LabelVariant } from "~/components/data-display";
+
+const DummyValueText = "Lorem ipsum sit amet";
+const DummyValueNumber = 1000;
+const DummyValueTextList = ["Lorem ipsum", "Dolor", "Sit amet"];
+const DummyValueNumberList = [200, 350, 1800];
+
+describe("ValueLabel", () => {
+  const renderComponent = ({
+    value,
+    placeholder,
+    variant,
+  }: {
+    value?: string | number | string[] | number[];
+    placeholder?: string;
+    variant?: LabelVariant | LabelVariant[];
+  }) => {
+    render(
+      <ValueLabel
+        label="Hello world"
+        value={value}
+        placeholder={placeholder}
+        variant={Array.isArray(variant) ? variant[0] : variant!}
+      />,
+    );
+  };
+
+  it("would render the label", () => {
+    renderComponent({ value: DummyValueText });
+
+    expect(screen.getByRole("label", { name: /hello world/i })).toBeInTheDocument();
```

Comment:
```suggestion
    expect(screen.getByRole("label", { name: /hello world/i })).toBeVisible();
```
I'm migrating from the `toBeInTheDocument` to `toBeVisible`. try to use the second one if is possible